{
  try {
  }
 catch (  E1|E2 ex) {
    final Exception b=ex;
  }
}
